nrf-hal-common/src/wdt.rs

```diff
@@ -202,7 +213,14 @@ where
     /// Is the watchdog active?
     #[inline(always)]
     pub fn is_active(&self) -> bool {
-        self.wdt.runstatus.read().runstatus().bit_is_set()
+        cfg_if! {
+            if #[cfg(feature = "9160")]
+            {
+                self.wdt.runstatus.read().runstatuswdt().bit_is_set()
```

Comment:
is the PAC wrong here or is this also called `runstatuswdt` in the datasheet?

Reply:
The PAC is right I believe. It appears like this in the datasheet: https://infocenter.nordicsemi.com/pdf/nRF9160_PS_v2.0.pdf - p.357